packages/react/src/profile/__tests__/useProfile.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { Profile } from '@lens-protocol/api-bindings';
+import {
+  mockLensApolloClient,
+  mockProfileResponse,
+  mockProfileFragment,
+  mockSources,
+  simulateNotAuthenticated,
+} from '@lens-protocol/api-bindings/mocks';
+import { waitFor } from '@testing-library/react';
+
+import { NotFoundError } from '../../NotFoundError';
+import { renderHookWithMocks } from '../../__helpers__/testing-library';
+import {
+  defaultMediaTransformsConfig,
+  mediaTransformConfigToQueryVariables,
+} from '../../mediaTransforms';
+import { useProfile, UseProfileArgs } from '../useProfile';
+
+function setupTestScenario({ profile, ...args }: UseProfileArgs & { profile: Profile | null }) {
+  const sources = mockSources();
+
+  return renderHookWithMocks(() => useProfile(args), {
+    mocks: {
+      sources,
```

Comment:
Can we remove `sources`?

Reply:
yes, I'm removing sources in another PR I'm working on right now, with useProfiles